test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightEntryAndFeedSerializerUndecalredTests.cs

```diff
@@ -235,6 +235,29 @@ public void WriteResourceCollectionUntypedProperty_WorksResourceInNestedCollecti
             Assert.Equal(result, actual);
         }
 
+        [Fact]
+        public void ODataBinaryStreamValue_Leaves_Stream_Not_Open_By_Default()
+        {
+            // With type name
+            // ODataBinaryStreamValue
+            var stream = new MemoryStream();
+            var writer = new BinaryWriter(stream);
+            writer.Write("1234567890");
+            writer.Flush();
+            stream.Position = 0;
+
+            var property = new ODataProperty
+            {
+                Name = "Data",
+                Value = new ODataBinaryStreamValue(stream)
+            };
+
+            string result = WriteDeclaredUntypedProperty(property);
+            Assert.Equal("{\"@odata.context\":\"http://www.sampletest.com/$metadata#serverEntitySet/$entity\",\"Data\":\"CjEyMzQ1Njc4OTA=\"}", result);
+
+            Assert.False(stream.CanRead);
+        }
+

```

Comment:
Is there a test that shows if you set `leaveOpen: true` that it won't automatically close the stream? If not, I think you should add that as well.

Reply:
It's there: https://github.com/OData/odata.net/blob/main/test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightValueSerializerTests.cs#L320